src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
```

Comment:
it's default value - no need to set it

Reply:
Cannot get what exactly can have a default value here. Please give broader suggestions.

Reply:
`400` is default value for [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#normal)